sdks/python/apache_beam/transforms/external_test.py

```diff
@@ -259,6 +259,33 @@ def test_unique_name(self):
     self.assertEqual(
         len(set(pcolls)), len(pcolls), msg='PCollection names are not unique.')
 
+  def test_external_transform_finder_non_leaf(self):
+    pipeline = beam.Pipeline()
+    _ = (
+        pipeline
+        | beam.Create(['a', 'b'])
+        | beam.ExternalTransform(
+            'beam:transforms:xlang:test:prefix',
+            ImplicitSchemaPayloadBuilder({'data': u'0'}),
+            expansion_service.ExpansionServiceServicer())
+        | beam.FlatMap(lambda x: x))
```

Comment:
`FlatMap` [needs](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/transforms/core.py#L1532) to return an iterable of elements. It might just work as it expands strings to characters and output is not important in this test.

Reply:
Changed to Map.